shark/iree_utils/vulkan_utils.py

```diff
@@ -139,7 +139,11 @@ def get_vulkan_target_triple(device_name):
         triple = f"rdna3-780m-{system_os}"
     elif all(x in device_name for x in ("AMD", "PRO", "W7900")):
         triple = f"rdna3-w7900-{system_os}"
-    elif any(x in device_name for x in ("AMD", "Radeon")):
+    elif "7600" in device_name:
+        triple = f"rdna3-7600-{system_os}"
+    elif "7700" in device_name:
+        triple = f"rdna3-7700-{system_os}"
```

Comment:
To be pedantic and fit in with the other rdna3 lines, should this be:

```python
    elif all(x in device_name for x in ("RX", "7600")):
        triple = f"rdna3-7600-{system_os}"
    elif all(x in device_name for x in ("RX", "7700")):
        triple = f"rdna3-7700-{system_os}"
```

...of course I don't either of those to check that is right.